packit_service/models.py

```diff
@@ -1012,6 +1012,15 @@ def get_all_by(
             )
             return query.all()
 
+    @classmethod
+    def get_all_by_commit(
+        cls, commit_sha: str
+    ) -> Optional[Iterable["CoprBuildTargetModel"]]:
+        """Returns all builds which matches a given commit sha"""
```

Comment:
```suggestion
        """Returns all builds that match a given commit sha"""
```
sorry for picking nits